src/enum.cr

```diff
@@ -352,7 +352,7 @@ struct Enum
   # Color.from_value?(2) # => Color::Blue
   # Color.from_value?(3) # => nil
   # ```
-  def self.from_value?(value : Int) : self?
```

Comment:
Isn't this going to make it no longer show up in the `Constructors` section in the docs? That's kind a bummer.

Reply:
Yeah, I agree. While I'll be happy to have contributed to Crystal if this PR is merged, I think I'd be even happier if someone found a better way to get `super` working in this case.

Reply:
Yes, that's an unfortunate consequence of this. It's still an acceptable short-term solution. And we should find a way to improve this, maybe with a compiler change or defining `Enum`'s class methods in `macro inherited`.